probe/core/src/main/java/org/jboss/weld/probe/InvocationMonitor.java

```diff
@@ -86,7 +86,7 @@
 
     @Intercepted
     @Inject
-    private Bean<?> interceptedBean;
+    private transient Bean<?> interceptedBean;
```

Comment:
-1. A built-in bean which is a passivation capable dependency (as defined by the spec) is injected.